cmd/nvidia-ctk/runtime/configure/configure.go

```diff
@@ -285,16 +305,40 @@ func (c *config) resolveConfigFilePath() string {
 		return c.configFilePath
 	}
 	switch c.runtime {
-	case "containerd":
+	case runtimeContainerd:
 		return defaultContainerdConfigFilePath
-	case "crio":
+	case runtimeCrio:
 		return defaultCrioConfigFilePath
-	case "docker":
+	case runtimeDocker:
 		return defaultDockerConfigFilePath
 	}
 	return ""
 }
 
+// resolveConfigSource returns the default config source or the user provided config source
+func (c *config) resolveConfigSource() (toml.Loader, error) {
+	switch c.configSource {
+	case configSourceCommand:
+		cmd := c.getConfigSourceCommand()
```

Comment:
How do we handle the (default) case where `docker` is the runtime and `command` is the source?

Reply:
Good question. I don't think there is a docker equivalent for `containerd config dump`. What if we just returned the `file` config source if the runtime was docker?